doc/md/examples/grammar.txt

```diff
@@ -69,7 +69,7 @@
 <typ_params_opt> ::= 
     <empty>
     '<' <list(<typ_bind>, ',')> '>'
-    '<' 'async' '>'
+    '<' 'async' (',' <typ_bind>)* '>'
```

Comment:
This means you allow the annotation on type constructor parameters too (see below), which we probably don't want.

Reply:
Fixed in c2e1a7eb9.